src/openvic-simulation/tech/Technology.cpp

```diff
@@ -100,47 +113,52 @@ bool TechnologyManager::load_technology_file_schools(ModifierManager const& modi
 bool TechnologyManager::load_technologies_file(ModifierManager const& modifier_manager, UnitManager const& unit_manager, BuildingManager const& building_manager, ast::NodeCPtr root) {
 	return expect_dictionary_reserve_length(technologies, [this, &modifier_manager, &unit_manager, &building_manager](std::string_view tech_key, ast::NodeCPtr tech_value) -> bool {
 		ModifierValue modifiers;
-		std::string_view area_identifier;
-		Technology::year_t year;
-		fixed_point_t cost;
+		TechnologyArea const* area = nullptr;
+		Technology::year_t year = 0;
```

Comment:
Why are these not just `Date::year_t`?

Reply:
This one is on me, sorry! I made my own alias in Technology because I forgot we had a centralised one.